src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -194,8 +194,9 @@ var ReactElementValidator = {
         false,
         'React.createElement: type should not be null, undefined, boolean, or ' +
           'number. It should be a string (for DOM elements) or a ReactClass ' +
-          '(for composite components).%s',
-        getDeclarationErrorAddendum()
+          '(for composite components).%s%s',
+        getDeclarationErrorAddendum(),
+        ReactComponentTreeHook.getCurrentStackAddendum()
```

Comment:
What if we reorder to call `ReactElement.createElement` first and pass element to `getCurrentStackAddendum(element)` to get stack addendum.

The output will be something like:

```
    in Unknown (at example.js:6)
    in Button (at example.js:12)
    in DangerButton (at example.js:25)
    in div (at example.js:24)
    in ExampleApplication (at example.js:35)
```

Is it good enough?

just curious about testing, the line number in output will be varied based on caller in test file. How can I test it?

```
-Warning: React.createElement: type should not be null, undefined, boolean, or number. It should be a string (for DOM elements) or a ReactClass (for composite components).
-    in Unknown (at ReactElementValidator-test.js:530)
+Warning: React.createElement: type should not be null, undefined, boolean, or number. It should be a string (for DOM elements) or a ReactClass (for composite components). 
+    in Unknown (at ReactElementValidator-test.js:537)
```

I'm new to react code base and would like to learn, Thanks. 😃